catalog/edf-methanesat-ee/description.md

```diff
@@ -0,0 +1,2 @@
+Dataset derived from data obtained from [MethaneAIR test flights](https://www.methanesat.org/project-updates/meet-methaneair-2/)
```

Comment:
Are these data still from air sensors or from MethaneSAT itself?

Reply:
I updated this text.